dev/tests/fixtures/docfx/Vision/V1.Word.yml

```diff
@@ -49,7 +49,7 @@ items:
         -
           id: '↳ bounding_box'
           var_type: '<xref uid="\Google\Cloud\Vision\V1\BoundingPoly">Google\Cloud\Vision\V1\BoundingPoly</xref>'
-          description: "The bounding box for the word. The vertices are in the order of top-left, top-right, bottom-right, bottom-left. When a rotation of the bounding box is detected the rotation is represented as around the top-left corner as defined when the text is read in the 'natural' orientation. For example: * when the text is horizontal it might look like: 0----1 | | 3----2 * when it's rotated 180 degrees around the top-left corner it becomes: 2----3 | | 1----0 and the vertex order will still be (0, 1, 2, 3)."
+          description: "The bounding box for the word. The vertices are in the order of top-left, top-right, bottom-right, bottom-left. When a rotation of the bounding box is detected the rotation is represented as around the top-left corner as defined when the text is read in the 'natural' orientation. For example: * * when the text is horizontal it might look like: 0----1 | | 3----2 * * when it's rotated 180 degrees around the top-left corner it becomes: 2----3 | | 1----0 and the vertex order will still be (0, 1, 2, 3)."
```

Comment:
Why is there two `*` here, is it something like created a list item and then surround the text with `*`?

Reply:
This matches the source protos

Reply:
it seems to look mostly good in the docs - https://cloud.google.com/php/docs/reference/cloud-vision/1.10.0/V1.Word#_Google_Cloud_Vision_V1_Word__getBoundingBox__